js/actors/grapple.js

```diff
@@ -49,13 +52,13 @@ class Grapple extends Actor {
         const diffX = targetX - this.getCenterX();
         const diffY = targetY - this.getCenterY();
         const mag = Math.sqrt(Math.pow(diffX, 2) + Math.pow(diffY, 2));
-        this.endX = Math.round(this.x + ((diffX / mag) * this.length));
-        this.endY = Math.round(this.y + ((diffY / mag) * this.length));
+        this.endX = Math.round(this.player.arm.x + ((diffX / mag) * this.length));
+        this.endY = Math.round(this.player.arm.y + ((diffY / mag) * this.length));
     }
 
     attach() {
         this.state = GrappleState.ATTACHED;
-        this.player.srcImage = "playerSpriteGrappled";
+        //this.player.srcImage = "playerSpriteGrappled";
```

Comment:
i know this is in a weird place, but its for the sake of only calling it once when a grapple attaches, rather than checking every loop in the player class. i'll think of a way to clean this up